packages/date-picker/test/keyboard-input.test.js

```diff
@@ -57,7 +57,8 @@ describe('keyboard', () => {
       expect(focusedDate().getFullYear()).to.equal(2000);
     });
 
-    it('should display focused date while overlay focused', async () => {
+    // FIXME: flaky test often failing locally due to scroll animation
+    it.skip('should display focused date while overlay focused', async () => {
```

Comment:
Naive question: Would it be possible to remove the scroll animation for this test, or was the test already flaky before this change?

Reply:
The test was already flaky before this change and stopped failing after #3798 where the problem was introduced.

I will try to patch `scrollDuration` in tests as part of #3904 since it would be easier to access overlay content.